modules/howtos/pages/subdocument-operations.adoc

```diff
@@ -2,24 +2,31 @@
-Sub-document operations are also atomic, allowing safe modifications to documents with built-in concurrency control.
+Sub-Document operations can be used to efficiently access and change parts of documents.
+
+include::partial$alpha-warning.adoc[]
+
+Sub-Document operations may be quicker and more network-efficient than _full-document_ operations such as _upsert_, _replace_ and _get_ because they only transmit the accessed sections of the document over the network.
+
+Sub-Document operations are also atomic, in that if one Sub-Document mutation fails then all will, allowing safe modifications to documents with built-in concurrency control.
 
 == Sub-documents
 
 Starting with Couchbase Server 4.5 you can atomically and efficiently update and retrieve _parts_ of a document.
-These parts are called _sub-documents_.
-While full-document retrievals retrieve the entire document and full document updates require sending the entire document, sub-document retrievals only retrieve relevant parts of a document and sub-document updates only require sending the updated portions of a document.
-You should use sub-document operations when you are modifying only portions of a document, and full-document operations when the contents of a document is to change significantly.
+These parts are called _Sub-Documents_.
+While full-document retrievals retrieve the entire document and full document updates require sending the entire document, Sub-Document retrievals only retrieve relevant parts of a document and Sub-Document updates only require sending the updated portions of a document.
+
+You should use Sub-Document operations when you are modifying only portions of a document, and full-document operations when the contents of a document is to change significantly.
 
-IMPORTANT: The sub-document operations described on this page are for _Key-Value_ requests only: they are not related to sub-document N1QL queries.
-(Sub-document N1QL queries are explained in the section xref:n1ql-query.adoc[Querying with N1QL].)
+IMPORTANT: The Sub-Document operations described on this page are for _Key-Value_ requests only: they are not related to Sub-Document N1QL queries.
+(Sub-Document N1QL queries are explained in the section xref:n1ql-queries-with-sdk.adoc[Querying with N1QL].)
 
-In order to use sub-document operations you need to specify a _path_ indicating the location of the sub-document.
-The _path_ follows N1QL syntax (see https://developer.couchbase.com/documentation/server/current/sdk/subdocument-operations.html#story-h2-12[below^], and xref:6.0@server:n1ql:n1ql-intro/queriesandresults.adoc[N1QL Queries and Results]).
+In order to use Sub-Document operations you need to specify a _path_ indicating the location of the Sub-Document.
+The _path_ follows <<Path syntax,N1QL syntax>>.
```

Comment:
```suggestion
The _path_ follows <<path-syntax,N1QL syntax>>.
```

Syntax for creating same page links https://docs.couchbase.com/home/contribute/ids.html#creating-a-link-to-an-element-in-the-same-page